.docker/Dockerfile

```diff
@@ -0,0 +1,55 @@
+FROM wordpress:5.0.3-fpm
+
+# Move WordPress files to Working Directory
+RUN mv /usr/src/wordpress /usr/src/public_html
+WORKDIR /usr/src/public_html
+
+# Install and configure nginx
+RUN apt-get update
+RUN apt-get install nginx -y
+COPY nginx.conf /etc/nginx/sites-enabled/wordcamp.conf
+COPY php-fpm.conf /usr/local/etc/php-fpm.d/zz-www.conf
+
+# Install and configure SSL
+COPY wordcamp.ssl.conf /var/wordcamp.ssl.conf
+RUN cd ~ && openssl req -x509 -nodes -days 365 -newkey rsa:2048 -keyout wordcamp.key -out wordcamp.crt -config /var/wordcamp.ssl.conf
+RUN cp ~/wordcamp.crt /etc/ssl/certs/wordcamp.crt
+RUN cp ~/wordcamp.key /etc/ssl/private/wordcamp.key
+
+# IAdd script to install plugin and themes
+ADD install-plugin-theme.sh /usr/src/public_html/install-plugin-theme.sh
+
+# Unfortunately themes and plugins cannot be installed as part of docker setup and have to be run manually.
+# This is because everything in wp-content folder will be overwritten when we mount this directory from volume.
+# RUN sh install-plugin-theme.sh
+
+
+# Install wkhtmltopdf for camptix invoices. See https://stackoverflow.com/a/38336153/1845153
```

Comment:
```suggestion
# Install `wkhtmltopdf` for WordCamp Docs, CampTix Invoices, etc. See https://stackoverflow.com/a/38336153/1845153.
```

Reply:
Fixed in https://github.com/WordPress/wordcamp.org/pull/113/commits/1e3bc2e5e0ebcc4643db21c0d98a69b94a51b6c8